### PR TITLE
[SDK] feat: update KVStoreClient [#1604]

### LINKED ISSUE #1604: SDK - Urls from kvstore
**Description**
This should be done in both TS and Python Sdks:
- Rename getUrl function from KVStore module to be more descriptive (Suggestion: getFileUrlAndVerifyHash)
- Rename setUrl function from KVStore module to be more descriptive (Suggestion: setFileUrlAndHash)
- Add a new function to specifically the public key: getPublicKey. It should pull the file, verify the hash and return the public key value. 

**Motivation**
Improve SDK usability

### PR DESCRIPTION
**Pull request by leric7**
## Description

Update KVStoreClient SDK more user friendly.

## Summary of changes

- Updated both Python & Typescript SDK KVStoreClient.
  - Renamed `getUrl` function to `getFileUrlAndVerifyHash`.
  - Renamed `setUrl` function to `setFileUrlAndHash`.
  - Added a new function to specifically the public key: `getPublicKey`. 
  - Updated tests.
  - Updated docs.
 
## Related issues
[Keywords for linking issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

Closes #1604

### CODE REVIEW

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/constants.py [hunk 1]
```diff
@@ -252,3 +252,9 @@ class Role(Enum):
 ARTIFACTS_FOLDER = os.path.join(os.path.dirname(os.path.dirname(__file__)), "artifacts")
 
 GAS_LIMIT = int(os.getenv("GAS_LIMIT", 4712388))
+
+
+class KVStoreKeys(Enum):
```
Comment (portuu3) [inline packages/sdk/python/human-protocol-sdk/human_protocol_sdk/constants.py:257]: bring other keys also

File: packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts [hunk 2]
```diff
@@ -370,4 +373,40 @@ export class KVStoreClient extends BaseEthersClient {
...
+    const publicKeyUrl = await this.getFileUrlAndVerifyHash(
+      address,
+      KVStoreKeys.publicKey
+    );
+
+    if (publicKeyUrl === '') {
+      return '';
+    }
+
+    const publicKey = await fetch(publicKeyUrl).then((res) => res.text());
```
Comment (portuu3) [inline packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts:408]: can you explain this? as far as I understand getFileUrlAndVerifyHash returns the url

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/constants.py (hunk 1 above)
Comment (leric7) [reply packages/sdk/python/human-protocol-sdk/human_protocol_sdk/constants.py:257]: Will do this in #1605

File: packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts (hunk 2 above)
Comment (leric7) [reply packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts:408]: Yeah, it returns URL. I think this getPublicKey function should return the content instead?

Comment (portuu3) [reply packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts:408]: I mean line 408. why you do this fetch?
Can't you do this?
`return publicKeyUrl;`

Comment (portuu3) [reply packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts:408]: Sorry, I understand now

**Review by portuu3 — APPROVED** [review]

**Pull request merged**